src/index.ts

```diff
@@ -1 +1,16 @@
-console.log('Hello, world!');
+import { getConfig } from './lib/getConfig';
+import { healthCheck } from './lib/healthCheck';
+
+const mainFunction = () => {
+  let index = 0;
+  while (true) {
+    index++;
+    const { url, webhook } = getConfig(index);
+    if (!url) break;
+
+    console.log(`[${index}] Checking ${url}...`);
+    healthCheck(url, webhook);
+  }
+};
+
+mainFunction();
```

Comment:
(minor, non-blocker) How_ about indicating an app is running?

Reply:
Thanks for the suggestion! Applied in [2630738](https://github.com/exxocism/node-healthz-prober/pull/2/commits/263073883d01cfca3cf7d7660fd369a08093e725)